component/file_cache/cache_policy.go

```diff
@@ -74,21 +75,28 @@ type cachePolicy interface {
 }
 
 // getUsagePercentage:  The current cache usage as a percentage of the maxSize
-func getUsagePercentage(path string, maxSize float64) float64 {
+func getUsagePercentage(path string, maxSize float64, useDu bool) float64 {
```

Comment:
again this can be acheived using function pointer instead of passing down the flags and then doing if-else check.